### PR TITLE
mariadb_1.1.4

### PR DESCRIPTION
**Pull request by avgra3**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by avgra3** [comment]
@help-python This is ready for review

**Comment by avgra3** [comment]
@conda-forge/staged-recipes all checks have passed. Pending your review/merge.

Thank you!

**Comment by avgra3** [comment]
@conda-forge/help-python this is ready for review.

Thank you!

File: recipes/mariadb/LICENSE [hunk 1]
```diff
@@ -0,0 +1,504 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (dopplershift) [inline recipes/mariadb/LICENSE:1]: This file is already included in the PyPI zip file, so you don't need to (and should not) include it here. Please remove.

File: recipes/mariadb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - mariadb
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.github.com/mariadb-corporation/mariadb-connector-python
+  summary: Python MariaDB extension
+  license: LGPL-2.1-only
```
Comment (dopplershift) [inline recipes/mariadb/meta.yaml:37]:
```suggestion
  license: LGPL-2.1-or-later
```

File: recipes/mariadb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mariadb-{{ version }}.zip
+  sha256: 73a0acbd2ace381ec1bcfc61ced7a79a519e66c02c24e12ae6d27baa03717876
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True
+  include_recipe: True
```
Comment (dopplershift) [inline recipes/mariadb/meta.yaml:17]: The skip here is telling the recipe not to build at all. Also, this recipe looks like it has an extension and can't be noarch?
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  number: 0
```

File: recipes/mariadb/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  host:
```
Comment (dopplershift) [inline recipes/mariadb/meta.yaml:20]: In order to compile:
```suggestion
  build:
    - {{ compiler('c') }}
  host:
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Let's see what happens when the recipe is actually built...

File: recipes/mariadb/LICENSE (hunk 1 above)
Comment (avgra3) [reply recipes/mariadb/LICENSE]: Removed License.

File: recipes/mariadb/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
```
Comment (dopplershift) [inline recipes/mariadb/meta.yaml:24]: With this not being noarch, the Python dependency should be unversioned:
```suggestion
    - python
    - pip
  run:
    - python
```

**Comment by carterbox** [comment]
Please mention the team again when you're ready.

**Pull request closed without merging**